include/cadmium/engine/pdevs_dynamic_runner.hpp

```diff
@@ -61,6 +61,8 @@ namespace cadmium {
             class runner {
                 TIME _next; //next scheduled event
 
+                bool progressBar = false;
```

Comment:
Use `snake_case` to comply with the style guide

Reply:
Done